### PR TITLE
[GS-1846] feat(st-table): Includes new field 'filterable' to apply filters 

### PR DESCRIPTION
**Pull request by jdiaz-stratio**
### Documentation
- [x] Add the issue in PR description
- [x] Have changelog updated
- [x] You fill the milestone value
- [x] You add some label
- [x] Have example running in demo app
- [ ] Review id on demo is the same of egeo folder

### Code
- [x] Pass Sass lint task
- [ ] Have qaTags

### Tests
- [x] Have at least 70% tests coverage

### CODE REVIEW

File: projects/egeo/src/lib/st-table/st-table.component.ts [hunk 1]
```diff
@@ -115,61 +126,96 @@ export class StTableComponent implements OnInit {
...
    private _selectedAll: boolean;
    private _hasHoverMenu: boolean = false;
 
+   constructor(private _cd: ChangeDetectorRef) {}
 
    ngOnInit(): void {
       this.tableClass = this.getClasses();
-   }
-
-   public onChangeOrder(field: StTableHeader): void {
```
Comment (emgaitan-stratio) [inline projects/egeo/src/lib/st-table/st-table.component.ts]: Check if these changes are needed

File: projects/egeo/src/lib/st-table/st-table.component.ts [hunk 2]
```diff
@@ -115,61 +126,96 @@ export class StTableComponent implements OnInit {
...
+         this.visibleFilter = -1;
+      } else {
+         this.visibleFilter = index;
+      }
+      this._cd.markForCheck();
    }
 
-   private isSortedByFieldAndDirection(field: StTableHeader, orderType: ORDER_TYPE): boolean {
-      return this.isSortedByField(field) && this.currentOrder.type === orderType;
+   public onChangeOrder(field: StTableHeader): void {
```
Comment (emgaitan-stratio) [inline projects/egeo/src/lib/st-table/st-table.component.ts]: Duplicated function

File: projects/egeo/src/lib/st-table/st-table.component.spec.ts [hunk 3]
```diff
@@ -119,6 +136,26 @@ describe('StTableComponent', () => {
       expect(headerItems[3].classList).toContain('st-table__header-item--sortable');
    });
 
+   it('only is displayed an arrow next to header of the field over which current filter is applied', () => {
```
Comment (emgaitan-stratio) [inline projects/egeo/src/lib/st-table/st-table.component.spec.ts]: Change text

Comment (jdiaz-stratio) [reply projects/egeo/src/lib/st-table/st-table.component.spec.ts]: Done

**Pull request merged**